gc/base/ParallelDispatcher.hpp

```diff
@@ -112,12 +112,11 @@ class MM_ParallelDispatcher : public MM_Dispatcher
 	
 	virtual void setThreadInitializationComplete(MM_EnvironmentBase *env);
 	
-	uintptr_t adjustThreadCount(uintptr_t maxThreadCount);
-	
 public:
 	virtual bool startUpThreads();
 	virtual void shutDownThreads();
 	
+	uintptr_t adjustThreadCount(uintptr_t maxThreadCount);
```

Comment:
Can you explain why this is now public?

Reply:
Evacuator controller uses this to force determination of gc thread count before dispatching slave threads. Metronome also needs to know gc thread count before dispatching its slave threads. If not called directly as evacuator controller is doing, the determination of gc thread count is not made until after controller starts its parallel task threads in scavenge(). 

There are other ways to do this (see metronome), but this is the most straightforward way and it works fine with MM_ParallelDispatcher/MM_ParallelTask.

Reply:
I am still confused as why the normal call to adjust does not work for Evacuator?

Reply:
Is there any more details?

Reply:
Is there any more details?